glide.yaml

```diff
@@ -93,7 +93,6 @@ import:
 - package: k8s.io/client-go
   version: v2.0.0
 - package: github.com/gambol99/go-marathon
-  version: d672c6fbb499596869d95146a26e7d0746c06c54
```

Comment:
Not sure I understand... why do we need the hash with go-marathon when we don't specify one with the vast majority of other dependencies?

Semantically, we shouldn't have to add it as we are not after a specific version: When we upgrade go-marathon, we tend to look for to latest.